src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
```

Comment:
I have updated #493 to reflect the outcome of our conversation. This will be addressed separately to keep this PR focussed.